Code/Source/Entity/MeshEntityManager.cpp

```diff
@@ -97,4 +102,9 @@ namespace RGL
             }
         }
     }
-} // namespace RGL
\ No newline at end of file
+
+    void MeshEntityManager::OnMaterialPropertiesUpdated([[maybe_unused]] const AZ::Render::MaterialAssignmentMap& materials)
+    {
+        AZ_Printf(__func__, "Material properties updated");
+    }
```

Comment:
Is there any purpose to this method, except of printing this information?

Reply:
It was there by mistake. I've removed it.